templates/docs/whats-new.md

```diff
@@ -52,805 +70,155 @@ When we add, make significant updates, or deprecate a component we update their
-      <td>
-        <span class="p-label--positive">New</span>
-      </td>
-      <td>2.36.0</td>
-      <td>We added a <code>.p-card__inner</code> element so that it's possible to have a mix of padded and not padded content within a card.</td>
-    </tr>
-    <tr>
-      <th><a href="/docs/base/forms#help-text">Form / Help text</a></th>
-      <td>
-        <span class="p-label--information">Updated</span>
-      </td>
-      <td>2.36.0</td>
-      <td>We added a class name to aligned help text with checkboxes or radio buttons</td>
-    </tr>
-    <!-- 2.35 -->
-    <tr>
-      <th><a href="/docs/base/forms#validation">Forms / Validation</a></th>
-      <td>
-        <span class="p-label--information">Updated</span>
-      </td>
-      <td>2.35.0</td>
-      <td>We've updated the styling of the form validation and required patterns</td>
-    </tr>
-    <!-- 2.34 -->
-    <tr>
-      <th><a href="/docs/base/forms#password-showhide">Forms / Password toggle</a></th>
-      <td>
-        <span class="p-label--positive">New</span>
-      </td>
-      <td>2.34.0</td>
-      <td>Password fields now have a show/hide toggle.</td>
-    </tr>
-    <tr>
-      <th><a href="/docs/patterns/lists#status">Lists / Crossed</a></th>
-      <td>
-        <span class="p-label--positive">New</span>
-      </td>
-      <td>2.34.0</td>
-      <td>We added a <code>is-crossed</code> modifier class for lists, to complement the existing <code>is-ticked</code> modifier.</td>
-    </tr>
-    <tr>
-      <th><a href="/docs/patterns/lists#status">Lists / Ticked</a></th>
-      <td>
-        <span class="p-label--information">Updated</span>
-      </td>
-      <td>2.34.0</td>
-      <td>We updated the color of the <code>is-ticked</code> icon on lists to <code>$color-positive</code>, where previously it was <code>$color-accent</code>.</td>
-    </tr>
-    <tr>
-      <th><a href="/docs/patterns/tabs#tab-buttons">Tabs / Tab Buttons</a></th>
-      <td>
-        <span class="p-label--positive">New</span>
-      </td>
-      <td>2.34.0</td>
-      <td>We added a new type of tab pattern, `p-tab-buttons`, which can display tabs like a group of buttons.</td>
-    </tr>
-    <!-- 2.33 -->
-    <tr>
-      <th><a href="/docs/patterns/links#skip-link">Links / Skip Link</a></th>
-      <td>
-        <span class="p-label--positive">New</span>
-      </td>
-      <td>2.33.0</td>
-      <td>We added a new link variant, <code>.p-link--skip</code>, that places a link offscreen and is revealed on focus.</td>
-    </tr>
-    <tr>
-      <th><a href="/docs/patterns/notification">Notifications</a></th>
-      <td>
-        <span class="p-label--information">Updated</span>
-      </td>
-      <td>2.33.0</td>
-      <td>Notifications have been updated with a new appearance, requiring a new HTML structure.</td>
-    </tr>
+    <!-- 3.00 -->
     <tr>
       <th><a href="/docs/patterns/notification#deprecated">Notifications structure</a></th>
```

Comment:
I just thought that it would be useful, instead of linking to removed sections or redirects, actually link to a migration guide section that talks about this change. 

So https://staging.vanillaframework.io/docs/migration-guide-to-v3#notifications in this case.
There should be relevant part of migration guide for all removed components.